src/taskUpdate.js

```diff
@@ -1,10 +1,23 @@
+import UserActions from './userActions.js';
+
+const userAction = new UserActions();
+
 export default class Tasks {
```

Comment:
After adding some tasks, marking some complete, then clearing the completed tasks, followed by adding some tasks again, all without reloading, there seems to be a duplication of indexes as you can see in the screenshot of the local storage. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/33268927/208238146-0ba5ab3a-1f23-462a-912c-9aa67465f65a.png)

Reply:
I have made changes. Thanks for your review.